tests/unittests/graphOptzTest.cpp

```diff
@@ -1047,25 +1047,26 @@ TEST_F(GraphOptz, concatElim) {
 
 // Check the transformation Concat(Reshape(x) * N) -> Reshape(Concat(x * N)).
 TEST_F(GraphOptz, concatReshapes) {
-  const size_t shape1[] = {10, 20};
-  const size_t shape2[] = {10, 1, 20};
+  const size_t shape1[] = {2, 5, 1, 20};
+  const size_t shape2[] = {10, 1, 2, 10};
   const size_t shape3[] = {5, 40};
   llvm::SmallVector<NodeValue, 10> inputs1;
   llvm::SmallVector<NodeValue, 10> inputs2;
   for (size_t i = 0; i < 10; i++) {
-    // 10 reshape nodes that transform from {10,20} to {10, 1, 20}.
+    const size_t shape2[] = {10, 1, 2, 10};
```

Comment:
Seems this was duplicated unnecessarily?

Reply:
thx!